COMMUNICATIONS.md

```diff
@@ -0,0 +1,83 @@
+# OpenSearch Dashboards Communication
+
+- [Overview](#overview)
+- [Slack](#slack)
+- [Forum](https://discuss.opendistrocommunity.dev/)
```

Comment:
Nit: I think we have https://forum.opensearch.org/

Reply:
nice catch - bad copy paste

Reply:
it's meant to link to the header section, anyway 😁